### PR TITLE
[IMP] payment_providers: remove IPN section from PayPal

### PR DESCRIPTION
**Pull request by anko-odoo**
Removing IPN mentions from the documentation.

Community PR: https://github.com/odoo/odoo/pull/104974

### CODE REVIEW

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Thanks for the update @anko-odoo :)
It's not required, but to help us double-check the content, it's good practice to explain the change (e.g., with a dev task, or the related community/enterprise PR)

Cheers!

File: content/applications/finance/payment_providers/paypal.rst [hunk 1]
```diff
@@ -16,8 +16,8 @@ open the :guilabel:`Website payments` menu.
    :alt: PayPal account menu
 
 .. important::
-   Note that for PayPal to work **in Odoo**, the options :ref:`Auto Return <paypal/auto-return>`,
-   :ref:`PDT <paypal/pdt>`, and :ref:`IPN <paypal/ipn>` **must** all be enabled.
+   Note that for PayPal to work **in Odoo**, the option :ref:`Auto Return <paypal/auto-return>`,
+   :ref:`PDT <paypal/pdt>` **must** be enabled.
```
[the remark this answers is not in the document]
Comment (anko-odoo) [reply content/applications/finance/payment_providers/paypal.rst:20]: Yes, you str right, good catch.

**Comment by jcs-odoo** [comment]
I just checked with @anko-odoo and the PR also needs to target saas-16.2 cc @toaa-odoo and @xpl-odoo ^^

**Review by jcs-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**